src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
+@import './utils/variables';
+
 body {
   margin: 0;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  border: 2px solid red;
 }
+
+.day {
+  box-sizing: border-box;
+  width: $days-width;
+  height: $days-height;
+  background-color: #eee;
+  border: 1px solid black;
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  color: black;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.month {
+  /* counter-reset: section; */
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  border: 3px solid black;
```

Comment:
Similar to the `body` element, inline border styles should be avoided. If this is a temporary style for debugging, it's better to place it in a separate stylesheet that is not included in production.